content/nomos/updates/2024-03-11.md

```diff
@@ -0,0 +1,68 @@
+
+## `cryptarchia`
+
+### `research`
+
+- We have updated the Cryptarchia specification with the update epoch stabilization schedule - [PR](https://github.com/logos-co/nomos-specs/pull/79).
+- Analysis of adversarial inference of relative stake: derived an equation that can be used to infer the Lagrange parameter in the maximum likelihood inference of relative stake - the details of the [analysis](https://www.overleaf.com/project/656dfacf4929b4a3d6d2ffe5).
+
+### `development`
+
+- Removed assumptions on Carnot being the consensus algorithm in the mempool: [PR](https://github.com/logos-co/nomos-node/pull/605).
+- Separate ledger and consensus to prepare for integration: [PR](https://github.com/logos-co/nomos-node/pull/606).
+
+## `data availability`
+
+### `research`
+
+- No current updates.
+
+### `development`
+
+- Finished RS core encode/decode: there was an issue with different FFT calls from different libraries that didn't work and took a while to debug. They use floating numbers, and when rounding or using a big set of operations, precision leads to errors - [PR](https://github.com/logos-co/nomos-specs/pull/75).
+- Implemented DA protocol encoder: [PR](https://github.com/logos-co/nomos-specs/pull/76).
+- Implemented DA protocol verifier: [PR](https://github.com/logos-co/nomos-specs/pull/78).
+- DA API mempool tests using a mock implementation [PR](https://github.com/logos-co/nomos-node/pull/604) - The previous PR defined an abstraction for verifying and filtering what to include in the generic mempool; this PR provides mock implementations for TX and Cert verification/filtering. WIP: DA API indexing for data blobs - adding an index to data blob in DA node when the certificate is observed in the block.
+
+## `coordination layer`
+
+### `research`
+
+- We have begun a couple of study documents in terms of the Coordination Layer: [What does it mean for an asset to be “inside” a Zone](https://www.notion.so/What-does-it-mean-for-an-asset-to-be-inside-a-Zone-a38134e2667e43fb80ccf6df0e3d1def) and [Illustrated guide to “Mutator Sets and their Application to Scalable Privacy](https://www.notion.so/Illustrated-guide-to-Mutator-Sets-and-their-Application-to-Scalable-Privacy-17236ccd86994a03b8a4541ea0341a6d). With these documents, we want to solve questions and challenges before delving into the design. The "Parallel Zero Knowledge Virtual Machine" paper has been reviewed, and a brief summary of GKR details has been shared.
+
+### `development`
+
+- Heavily in research, no development updates.
+
+## `miscellaneous`
+
+- Nomos has a new [HackMD](https://hackmd.io/@Nomos) account - our team will be publishing various notes on it - mostly scientific in nature.
+- Blog to be released this week. Stay tuned on our website.
+- The plan for the rest of the year, objectives, required resources, planning, risk assessment, approach to execution will be shared soon.
```

Comment:
I don't know if it's going to be shared of not - better not to mention it just in case